api/src/apiTest/java/com/ford/labs/retroquest/api/ThoughtApiTest.java

```diff
@@ -419,14 +336,13 @@ public void canDeleteAllThoughtsWithWebsockets() throws Exception {
         headers.add("Authorization", "Bearer " + jwt);
 
         StompSession session = stompClient
-                .connect(websocketUri, new WebSocketHttpHeaders() {}, headers, new StompSessionHandlerAdapter() {})
+                .connect(websocketUri, new WebSocketHttpHeaders() {
+                }, headers, new StompSessionHandlerAdapter() {
+                })
                 .get(1, SECONDS);
         session.subscribe(thoughtSubscribeEndpoint, new DefaultStompFrameHandler());
         String thoughJsonBody = "{\"message\":\"Message\"}";
         session.send(thoughtEndpoint + "/create", thoughJsonBody.getBytes());
-
-        Thought thought = getLatestThoughtInQueue();
```

Comment:
This removal needs to be added back into the test, because the create response is stuck in the queue and is being improperly parsed as a Long, because of the `final Long returnVal = getLatestIdInQueue();`